### ISSUE TITLE
Improve facial recognition

### ISSUE THREAD

**Issue by AmrMKayid** [AmrMKayid/whoami#3]
Using tensorflow and keras to improve the accuracy
